src/site/_data/authorsData.json

```diff
@@ -1688,5 +1688,10 @@
     "homepage": "https://davidea.st",
     "twitter": "_davideast",
     "image": "image/vS06HQ1YTsbMKSFTIPl2iogUQP73/S8CN1AfcJpoUJvYqrfeT.jpeg"
+  },
+  "superhighfives": {
+    "homepage": "https://t.co/IGOafTKR29",
```

Comment:
Can this just be a direct link to https://charliegleason.com/?

Reply:
def, looks like a copy n pasta error 👍🏻